lib/matplotlib/colors.py

```diff
@@ -2016,8 +2017,9 @@ class SegmentedBivarColormap(BivarColormap):
 
     Parameters
     ----------
-    patch : np.array of shape (k, k, 3)
-        This patch gets supersampled to a lut of shape (N, M, 4).
+    patch : np.array
+        Patch is required to have a shape (k, l, 3), and will get supersampled
```

Comment:
Is this supposed to be `(k, k, 3)` not `(k, l, 3)`? Sorry, still not sure what `k` is; is it saying it must be square?

Reply:
It used to be, but I updated the code so that it also works with non-square inputs.